src/crypto/slow-hash.c

```diff
@@ -1464,6 +1526,10 @@ STATIC INLINE void xor_blocks(uint8_t* a, const uint8_t* b)
 
 void cn_slow_hash(const void *data, size_t length, char *hash, int variant, int prehashed, uint64_t height)
 {
+    if (variant >= 6) {
+        rx_slow_hash(data, length, hash);
+        return;
+    }
```

Comment:
All these 4 of those checks should really be moved to just one in get_block_longhash. Then the randomx code can be kept in its file AFAICT rather than a CN bodge.

Reply:
Moved to rx-slow-hash.c